docs/source/requirements.rst

```diff
@@ -6,14 +6,14 @@ Requirements
 ============
 
 A control node is any machine with Ansible installed. From the control node,
-you can run commands and playbooks from a laptop, desktop, or server machine.
-However, you cannot run **IBM z/OS core collection** on a Windows machine.
+you can run commands and playbooks from a laptop, desktop, or server.
+However, you cannot run **IBM z/OS core collection** on a Windows system.
 
```

Comment:
please review the use of the windows server/system etc. IBM word usage discourages using machines